### ISSUE TITLE
about the webhook url

### ISSUE THREAD

**Issue by lvjincan** [payeezy/payeezy_apple_pay#1]
Hello Sir/Madam
in the page 
https://github.com/payeezy/payeezy_apple_pay

setp6 . If provided, Payeezy sends out a transaction receipt to the merchant URL (Via a Webhook)

So i want provide a the merchant website URL, and the URL want to known if Payeezy received money about one transaction, and then merchant website will set the order status to 'paid'.

So Where is the guide to developer the merchant web URL?
What is the detail about a transaction receipt?

thank you very much.

Cecelia LV

**Comment by lvjincan** [comment]
hello Sir/Madam

I found some informaiton in the link : https://developer.payeezy.com/content/what-webhook-url
Re: What is webhook URL?
Interacting with APIs like Payeezy often suffers from two important problems:
1. Services not directly responsible for making an API request and  may still need to know the response.
2. Some events, like disputed charges and many recurring billing events, are not the result of a direct API request.

Webhooks solve these problems by letting you register a URL that we will POST anytime an event happens in your account. When the event occurs, for example when a successful charge is made in your account, Payeezy creates an event object. This object contains all the relevant information, including the type of event and the data associated with that event. Payeezy then sends an HTTP POST request with the event object to any URLs in your account's webhook settings.

So where can I found the detail description of HTTP POST request which Payeezy send.

thank you.

Cecelia LV
